### ISSUE TITLE
Select: `aria-invalid` attribute missing in non-editable mode

### ISSUE THREAD

**Issue by arpi17** [primefaces/primevue#6813]
### Describe the bug

In non-editable mode the `Select` component doesn't have the `aria-invalid="true"` attribute when the `invalid` property is set on it. In editable mode the attribute is [correctly set](https://github.com/primefaces/primevue/blob/3c426a1386556ecc56241ec429947e448e4c3016/packages/primevue/src/select/Select.vue#L22) when setting the `invalid` property.

This attribute would be nice to have also in non-editable mode since it not just improves screen reader support but also enables styling the component in pure unstyled mode (since now in pure unstyled mode setting the `invalid` property in non-editable mode does not have any effect on the component)

### Reproducer

https://stackblitz.com/edit/primevue-4-vite-issue-template-trdk6i?file=src%2FApp.vue

### PrimeVue version

4.2.2

### Vue version

3.x

### Language

ALL

### Build / Runtime

Vue CLI App
